CHANGELOG.rst

```diff
@@ -30,6 +30,43 @@ History
 
 * **Breaking Change**:  Change signature of hook ``update_internal_deposition_layer``. Now, instead of the deposition thickness, it is returned the phase index of phase being deposited and the thickness variation rate.
 * **Breaking Change**:  Change signature of hook ``calculate_relative_emulsion_viscosity`` and function ``get_relative_emulsion_viscosity``. Indices of continuous and dispersed fields were removed.
+* **Breaking Change**: Change in ``PhysicsDescription``: split emulsion model into relative viscosity, droplet size, and inversion point models and a flag to activate the emulsion models.
+
+* Examples on how to update previous .alfacase files with emulsion model:
+
+  - Relative viscosity model:
```

Comment:
Formatting was still broken: you used tabs instead of spaces, and the code needed a bit more indentation. 👍

Reply:
Thanks for fixing it ;)